libs/core/src/components/pds-text/pds-text.tsx

```diff
@@ -23,7 +23,8 @@ export class PdsText {
   | 'danger'
   | 'info'
   | 'success'
-  | 'warning';
+  | 'warning'
+  | string;
```

Comment:
This could be written as the suggestion below. This is possible because, in your setColor function, you are predefining a list of look values. 

```suggestion
@Prop() color?: string
```

Reply:
Thanks for the help on refactoring this